app/Filters/AppointmentFilters.php

```diff
@@ -18,7 +21,30 @@ public function status ($value = null)
     public function time($time)
     {
         $from = $time;
-        $to = $time + (request()->duration * 60000);
-        return $this->builder->whereBetween(DB::raw('time + (duration * 60000)'), [$from, $to])->where('status', AppointmentStatus::APPROVED);
+        $to = $time + (request()->duration * 60);
+        return $this->builder->whereBetween(DB::raw('time + (duration * 60)'), [$from, $to])->where('status', AppointmentStatus::APPROVED);
+    }
+
+    public function currentAppointment()
+    {
+        if (env('DB_CONNECTION') === 'sqlite') {
+            $results = DB::connection('sqlite')->select('select * from appointments WHERE
+            strftime("%s",time) <=  strftime("%s","now") AND status = 2');
+            //       AND(strftime("%s",time) + (duration *60)) >= strftime("%s","now")
+            if ($results == []) {
+                abort(Response::HTTP_BAD_REQUEST);
+            } else {
+                return $results;
+            }
```

Comment:
- No responses are returned from the filters
- In case there's no current appointment for the patient, just return empty data response